ansible/inventory.yml

```diff
@@ -74,6 +74,8 @@ hosts:
 
       - azure:
           win2012r2-x64-1: {ip: 13.68.134.204, user: adoptopenjdk}
+          win2012r2-x64-2: {ip: 40.121.164.56, user: adoptopenjdk}
```

Comment:
Is that IP correct?

Reply:
It's definitely the one @tellison gave me for that machine and does work - so it's connecting to some windows box somewhere in the cloud ;-)